members.js

```diff
@@ -1,81 +1,1617 @@
+        "role": "member"
+        },
+        {
+        "id" : "259611736",
+        "name" : "María Luisa Vides",
+        "high" : "https://secure.meetupstatic.com/photos/member/b/6/3/7/highres_279166647.jpeg",
+        "photo" : "https://secure.meetupstatic.com/photos/member/b/6/3/7/member_279166647.jpeg",
+        "thumb" : "https://secure.meetupstatic.com/photos/member/b/6/3/7/thumb_279166647.jpeg",
+        "role": "member"
+        },
+        {
+        "id" : "259545662",
+        "name" : "melody Durán ",
+        "high" : "https://secure.meetupstatic.com/photos/member/4/f/5/1/highres_279140305.jpeg",
+        "photo" : "https://secure.meetupstatic.com/photos/member/4/f/5/1/member_279140305.jpeg",
+        "thumb" : "https://secure.meetupstatic.com/photos/member/4/f/5/1/thumb_279140305.jpeg",
+        "role": "member"
+        },
+        {
+        "id" : "203363128",
+        "name" : "Ricardo Serje",
+        "high" : "https://secure.meetupstatic.com/photos/member/5/6/b/9/highres_255682201.jpeg",
+        "photo" : "https://secure.meetupstatic.com/photos/member/5/6/b/9/member_255682201.jpeg",
+        "thumb" : "https://secure.meetupstatic.com/photos/member/5/6/b/9/thumb_255682201.jpeg",
+        "role": "member"
+        },
+        {
+        "id" : "259241777",
+        "name" : "Francisco Viana",
+        "high" : "https://secure.meetupstatic.com/photos/member/5/7/0/1/highres_279022273.jpeg",
+        "photo" : "https://secure.meetupstatic.com/photos/member/5/7/0/1/member_279022273.jpeg",
+        "thumb" : "https://secure.meetupstatic.com/photos/member/5/7/0/1/thumb_279022273.jpeg",
+        "role": "member"
+        },
+        {
+        "id" : "130002012",
+        "name" : "Jhonnatan",
+        "high" : "https://secure.meetupstatic.com/photos/member/a/9/8/9/highres_278923401.jpeg",
+        "photo" : "https://secure.meetupstatic.com/photos/member/a/9/8/9/member_278923401.jpeg",
+        "thumb" : "https://secure.meetupstatic.com/photos/member/a/9/8/9/thumb_278923401.jpeg",
+        "role": "member"
+        },
+        {
+        "id" : "258956006",
+        "name" : "Karen Barrera",
+        "high" : "https://secure.meetupstatic.com/photos/member/6/2/f/f/highres_278905343.jpeg",
+        "photo" : "https://secure.meetupstatic.com/photos/member/6/2/f/f/member_278905343.jpeg",
+        "thumb" : "https://secure.meetupstatic.com/photos/member/6/2/f/f/thumb_278905343.jpeg",
+        "role": "member"
+        },
+        {
+        "id" : "173204422",
+        "name" : "Boniek Berdugo",
+        "high" : "https://secure.meetupstatic.com/photos/member/2/8/c/a/highres_228790442.jpeg",
+        "photo" : "https://secure.meetupstatic.com/photos/member/2/8/c/a/member_228790442.jpeg",
+        "thumb" : "https://secure.meetupstatic.com/photos/member/2/8/c/a/thumb_228790442.jpeg",
+        "role": "member"
+        },
+        {
+        "id" : "231222084",
+       
```

Comment:
Para este padacillo de codigo yo usaría template literals. Algo asi como : 
```suggestion
export default function addMember(role, photo, name) {
  return `
    <li class="${role}-flex-item">
      <img src="${photo}" alt="Photo of ${name}" class="member-photo">
      <div class="member-name">${name}</div>
    </li>
  `;
}
```
y luego lo exportaria en `membersTemplate`. En el link un breve ejemplo
https://codesandbox.io/s/xp8wll73kw